checks/ibm_mq_queues

```diff
@@ -154,7 +164,7 @@ def ibm_mq_last_age(mq_date, mq_time, agent_timestamp, label, key, params):
     input_time = dateutil.parser.parse(mq_datetime, default=agent_timestamp)
     age = (agent_timestamp - input_time).total_seconds()
     return check_levels(age,
-                        None,
+                        key,
```

Comment:
This is a powerful change that results in metrics being created (I know you know ;-)). It should be mentioned in the commit message, I think.

Reply:
Yes, I am aware :-)

Am I allowed to `git commit --amend` and then `git push --force` on this pull requested that you started to review?

Reply:
Absolutely

Reply:
After reading your comments about metrics and thinking more about the metrics I will make this change in another pull request.